partials/_v2_transition_banner.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="border border-violet-600 bg-violet-50 rounded-l p-4 my-4 text-base text-navy">
+We're starting to roll out our new Apps V2 platform to new organizations. Where there's any difference in instructions between Apps V2 and the original Nomad-orchestrated app platform, we'll supply the Nomad-specific variant either inline or at the end of the doc.
+<br><br>
+You can check if your org defaults to Apps V2 deployments with `fly orgs apps-v2 show <org-slug>`. Get your organization slug using `fly orgs list`.
```

Comment:
One more minor thought here! 

When I run this `fly orgs apps-v2 show...`, I get "Apps V2 default on: false", if there is a doc on setting this to true, I think it might also be helpful to add a link to that here.

Of course completely minor thought! since this banner I think might be removed after all apps move to v2

Reply:
Yeah, that's the idea. There's a forum post for early adopters to try that, but we don't want people going and changing back to Nomad in general.

Also, the banner's pretty big already :)